src/components/accession2/edit/Accession2EditModal.tsx

```diff
@@ -30,6 +33,10 @@ export default function Accession2EditModal(props: Accession2EditModalProps): JS
   const [record, setRecord, onChange] = useForm(accession);
   const [validateFields, setValidateFields] = useState<boolean>(false);
   const snackbar = useSnackbar();
+  const { selectedOrganization } = useOrganization();
+  const selectedSeedBank = getSeedBank(selectedOrganization, record.facilityId);
+  const tz = useLocationTimeZone().get(selectedSeedBank);
```

Comment:
use isEnabled flag here.. or maybe in the line below and initialize timeZone to undefined..
i think maybe it's ok to leave it as-is. just calling it out

Reply:
Yes, I will add it just in case